src/components/ResultContainer.tsx

```diff
@@ -156,14 +157,28 @@ function ResultContainer({ result }: { result: Result[] | null }) {
   }
 
   function renderResults() {
-    if (result === null) {
+    if (response === null) {
       return (
         <Typography variant="h5" data-cy="default-result-container-view" className="text-gray-500">
           Click &apos;Submit Query&apos; for results
         </Typography>
       );
     }
-    if (result.length === 0) {
+
+    if (response.nodes_response_status === 'failure') {
+      return (
+        <div data-cy="failed-result-container-view">
+          <Typography variant="h5" className="text-red-500">
```

Comment:
This is more of a personal preference, I think using the default black color is better here as opposed to red especially since we're already displaying a red toast in case of failure.

Reply:
Agree, was considering this as well because red looks scary!